portal-test/src/com/liferay/portal/kernel/test/rule/AggregateTestRule.java

```diff
@@ -63,7 +63,7 @@ public Statement apply(Statement statement, Description description) {
 		CodeCoverageAssertor.class.getName(), NewEnvTestRule.class.getName(),
 		AssumeTestRule.class.getName(),
 		"com.liferay.portal.test.rule.LiferayIntegrationTestRule",
-		"com.liferay.portal.test.rule.HypersonicServerClassTestRule",
+		"com.liferay.counter.test.HypersonicServerClassTestRule",
```

Comment:
@dantewang @yuhai 
Can you change the test to remove this line here? The simplest way is to avoid auto sort of test rule.

Thanks.
Tina.